components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/resources/publisher-api.yaml

```
@@ -16411,11 +16411,19 @@ components:
             name of Scope
           example: apim:api_view
         usedApiList:
+          deprecated: true
           type: array
           description: |
-            API list which have used the shared scope
+            API list which have used the shared scope. [DEPRECATED] Use `usages` instead. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)